app/views/user/manage.phtml

```diff
@@ -26,6 +26,13 @@
 			</div>
 		</div>
 
+		<div class="form-group">
+			<label for="new_user_is_admin" class="group-name"><?= _t('admin.user.is_admin') ?></label>
+			<div class="group-controls">
+				<input type="checkbox" name="new_user_is_admin" id="new_user_is_admin">
+			</div>
```

Comment:
Maybe we can put the label at the right of the checkbox, as it is for most of the other checkboxes?

Reply:
Ok, I'll work on that

Reply:
Done